public/src/lobby/Set.jsx

```diff
@@ -17,8 +17,9 @@ const Set = ({ index, selectedSet, type }) => {
         <option value={code} key={code}>{name}</option>
       );
     });
+    const label = setType.split('_').reduce((label, word) => `${label} ${word.charAt(0).toUpperCase()+word.slice(1).toLowerCase()}`, '')
```

Comment:
I swear I wrote a function to do this already elsewhere

Reply:
Found it in StartPanel.jsx line 7. 

Should I go ahead and make this a utils function?

Reply:
Yeah it should be a util function